locale/en-us/vocabulary/condition/do-i-need-an-umbrella.intent

```diff
@@ -1,4 +1,36 @@
-should I bring an umbrella
-do I need an umbrella
-should I bring a rain coat
-do I need a rain jacket
+will it rain today
+is there rain in the forecast
+do I need to prepare for rain
```

Comment:
just as an aside. work in "en-us" will trigger autotranslate.
Before you merge, please let me clean dialogs with langs that have wrong entities, so those can also be auto translated.